prod/deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+source /home/openfisca/virtualenvs/new-api/bin/activate
```

Comment:
Arg no, the deploy script is run by a deployment user, who is not `openfisca`. We cannot use `$HOME`.